workorder-job-repeater.lua

```diff
@@ -0,0 +1,219 @@
+-- simulates workshop's "repeat" for infinite work orders
+
+local function print_help()
+    print[====[
+
+workorder-job-repeater
```

Comment:
* _difference with check daily_: yes, this is different, because there is no time to lazy around between jobs until manager comes around to do the daily check
* _name_: I think `workorder` should be the prefix, to group workorder-related scripts together. `workorder-faster-infinity`?
* _workflow_: f.e. when you don't use workflow :) (I actually completely forgot about it). The main difference is, since it's a work order, it isn't bound to a workshop (and an infinite work order automatically gets assigned to multiple workshops). I.e. this script acts on a more global level. I'll try to improve the help in this direction.
* _differences from workflow_: you are correct, the job disappears for the day (or a bit longer for some reason) thus only producing a single cancellation per day

Reply:
Now that I think about what to write regarding `workflow` similarities, I notice it will only add confusion.

`workflow` _protects all **repeat jobs** from removal_. But if you are using work orders, you wouldn't notice this issue at all (like I didn't), because work orders themselves don't get cancelled.